### ISSUE TITLE
Test failure for test-methods:12-19

### ISSUE THREAD

**Issue by AshesITR** [r-lib/lintr#479]
While working on my PR I encountered a test failure on my local machine:

https://github.com/jimhester/lintr/blob/edb3b12aac25d57759fb20dcd0171ba510d6242f/tests/testthat/test-methods.R#L12-L19

Might be a windows thing, but here are the failures.
I'm not sure if the test is bad or the code, any guidance?

```
--------------------------------------------------------------------------------
test-methods.R:14: failure: it returns the input trimmed to the last full lint if one exists within the max
trim_output(t1, max = 200) not equal to substr(t1, 1, 195).
1/1 mismatches
x[1]: "[R/AES.R:30:34:](https://github.com///blob//R/AES.R#L30) *style:* **Put s
x[1]: paces around all infix operators.**\r\n```r\r\n            for (i in seq_l
x[1]: en(len/16)) {\r\n                                ~^~\r\n```\r\n"
y[1]: "[R/AES.R:30:34:](https://github.com///blob//R/AES.R#L30) *style:* **Put s
y[1]: paces around all infix operators.**\r\n```r\r\n            for (i in seq_l
y[1]: en(len/16)) {\r\n                                ~^~\r\n"

test-methods.R:16: failure: it returns the input trimmed to the last full lint if one exists within the max
trim_output(t1, max = 400) not equal to substr(t1, 1, 380).
1/1 mismatches
x[1]: "[R/AES.R:30:34:](https://github.com///blob//R/AES.R#L30) *style:* **Put s
x[1]: paces around all infix operators.**\r\n```r\r\n            for (i in seq_l
x[1]: en(len/16)) {\r\n                                ~^~\r\n```\r\n[R/AES.R:31
x[1]: :26:](https://github.com///blob//R/AES.R#L31) *style:* **Put spaces around
x[1]:  all infix operators.**\r\n```r\r\n                ind <- (i-1)*16 + 1:...
y[1]: "[R/AES.R:30:34:](https://github.com///blob//R/AES.R#L30) *style:* **Put s
y[1]: paces around all infix operators.**\r\n```r\r\n            for (i in seq_l
y[1]: en(len/16)) {\r\n                                ~^~\r\n```\r\n[R/AES.R:31
y[1]: :26:](https://github.com///blob//R/AES.R#L31) *style:* **Put spaces around
y[1]:  all infix operators.**\r\n```r\r\n                ind <- (i-1)*16 + 1:...

test-methods.R:18: failure: it returns the input trimmed to the last full lint if one exists within the max
trim_output(t1, max = 2000) not equal to substr(t1, 1, 1930).
1/1 mismatches
x[1]: "[R/AES.R:30:34:](https://github.com///blob//R/AES.R#L30) *style:* **Put s
x[1]: paces around all infix operators.**\r\n```r\r\n            for (i in seq_l
x[1]: en(len/16)) {\r\n                                ~^~\r\n```\r\n[R/AES.R:31
x[1]: :26:](https://github.com///blob//R/AES.R#L31) *style:* **Put spaces around
x[1]:  all infix operators.**\r\n```r\r\n                ind <- (i-1)*16 + 1:...
y[1]: "[R/AES.R:30:34:](https://github.com///blob//R/AES.R#L30) *style:* **Put s
y[1]: paces around all infix operators.**\r\n```r\r\n            for (i in seq_l
y[1]: en(len/16)) {\r\n                                ~^~\r\n```\r\n[R/AES.R:31
y[1]: :26:](https://github.com///blob//R/AES.R#L31) *style:* **Put spaces around
y[1]:  all infix operators.**\r\n```r\r\n                ind <- (i-1)*16 + 1:...
--------------------------------------------------------------------------------
```

**Comment by MichaelChirico** [comment]
@AshesITR  is this still affecting you?

**Comment by AshesITR** [comment]
I've switched to linux as my main OS on my private machine in the mean time. I remember the problem was lintr's code not being on C:.

Let me boot up Windows later and see if I can still reproduce it.

**Comment by AshesITR** [comment]
Can't reproduce ad-hoc. Closing for now.
